### PR TITLE
Kalai linkedlist v2

### PR DESCRIPTION
**Pull request by 31673**

### CODE REVIEW

File: linkedlist.c [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		count ++;
+		newNode = newNode->next;
+	}
+	printf("\nTotal nodes:%d\n", count);
+}
+
+int main(){
+    int choice;
+    void (*ptr [2])()={Insertion_Beginning, Display};
+    while(1)
```
Comment (tklogesh) [inline linkedlist.c:42]: Conversion related to [PR-8](https://github.com/preeja398/Training/pull/8#discussion_r1040882240), Infinite loop is not the solution and its a bug. you need to break the loop right?

Comment (31673) [reply linkedlist.c:42]: yes ,but  How can I that

File: linkedlist.c [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    {
+	    printf("\n0.Insertion_Beginning\n1.Display\n2.Exit\n");
```
Comment (P-Prajwal) [inline linkedlist.c:44]: Please remove "\n" from beginning, already suggested in previous PR: 
https://github.com/preeja398/Training/pull/8#discussion_r1040925279

File: linkedlist.c (hunk 1 above)
Comment (tklogesh) [reply linkedlist.c:42]: just give break; after printf("\n******Enter the valid choice******\n");

Comment (P-Prajwal) [reply linkedlist.c:42]: @tklogesh  the while loop will break when 2 is pressed, so its not infinite anymore. Is this what you were referring to or is it something else, dint get you?

Comment (P-Prajwal) [reply linkedlist.c:42]: @tklogesh but that will break loop without even taking inputs, seems ok to me the way it is now, user can break loop incase dont want to display/insert elements.  If you were suggesting some other method/better way of doing this dint get you.

File: linkedlist.c (hunk 2 above)
Comment (31673) [reply linkedlist.c:44]: ok

File: linkedlist.c (hunk 1 above)
Comment (tklogesh) [reply linkedlist.c:42]: In that case, if the input is non-integer then break; don't want to disturb this else part.

Comment (tklogesh) [reply linkedlist.c:42]: In that case,  you need to check weather the given input is integer or non integer. If its a non-integer create a condition/func this should be start point. For function/condition return printf with Enter valid choice loop only once with Boolean value. Once the boolean is T/F depends on the choice you make stop that loop

**Pull request closed without merging**